src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add: function(num) {
+      this.result += num;
+
+      return this;
```

Comment:
```suggestion
```
There is also no need to return this in operations. It's needed only in reset and operate methods